Sources/LeafKit/LeafSyntax/LeafSyntax.swift

```diff
@@ -511,28 +513,6 @@ extension Syntax {
         }
     }
 
-    public struct Variable {
-        public let path: [String]
```

Comment:
`Syntax.Variable` has been entirely removed as it's basically pointless - replaced by instantiating top level syntax objects as `.expression([.variable])` where the expression is a single value.